main.go

```diff
@@ -32,13 +62,33 @@ func main() {
 
 	log.Infof("\nRunning go test...")
 
+	packageCodeCoveragePth, err := packageCodeCoveragePath()
+	if err != nil {
+		failf(err.Error())
```

Comment:
if we run into an issue, it could be hard to investigate where the error came from.

Reply:
i designed the method to report a proper error message. no need for wrapping the error

Reply:
now, yes